### PR TITLE
German language created

### PR DESCRIPTION
**Pull request by buster-one**
All lines were translated to german, except one i could not find in the po file.

`For each link, add a Wallabag icon to save it in your instance.`

A bit of fine tuning is certainly still needed. Change requests are welcome. In my opinion according to the current status, it can be published.

### CODE REVIEW

**Comment by ArthurHoaro** [comment]
That's great! Thank you! 

I don't know what happened with that line, it should have been generated from the plugin administration page, just like for the other plugins... Did you use the `generate_templates` script?

How do we want to process for future new translation strings:

  * Do we just wait for eventual language update PR once in a while?
  * Do you want to be the maintainer of German translation, and be notified of new translations?

File: inc/languages/de/LC_MESSAGES/shaarli.po [hunk 1]
```diff
@@ -0,0 +1,1408 @@
...
+msgstr "Plugin"
+
+#: tmp/tools.b91ef64efc3688266305ea9b42e5017e.rtpl.php:175
+msgid ""
+"Drag this link to your bookmarks toolbar, or right-click it and choose "
+"Bookmark This Link"
+msgstr ""
+"Ziehe diese Link in deine Lesezeichen-Symbolleiste oder klicke mit der "
+"rechten Maustaste darauf und wähle \"Speichere diesen Link als Lesezeichen\""
+
```
Comment (virtualtam) [inline inc/languages/de/LC_MESSAGES/shaarli.po]: lines 1314 through 1408 can be removed

File: inc/languages/de/LC_MESSAGES/shaarli.po [hunk 2]
```diff
@@ -0,0 +1,1408 @@
+msgid ""
+msgstr ""
+"Project-Id-Version: Shaarli\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2018-03-31 09:09+0200\n"
+"PO-Revision-Date: 2018-03-31 09:12+0200\n"
+"Last-Translator: \n"
```
Comment (virtualtam) [inline inc/languages/de/LC_MESSAGES/shaarli.po]: feel free to add your contact information here :)

File: inc/languages/de/LC_MESSAGES/shaarli.po [hunk 3]
```diff
@@ -0,0 +1,1408 @@
...
+msgid_plural "The tag was removed from %d links."
+msgstr[0] "Der Tag wurde aus dem Link %d entfernt."
+msgstr[1] "Der Tag wurde aus den Links %d entfernt."
+
+#: index.php:1173
+#, php-format
+msgid "The tag was renamed in %d link."
+msgid_plural "The tag was renamed in %d links."
+msgstr[0] "Der Tage wurde im Link %d umbenannt."
+msgstr[1] "Der Tage wurde in den  Links %d umbenannt."
```
Comment (virtualtam) [inline inc/languages/de/LC_MESSAGES/shaarli.po]: extra space between `den` and `Links`

**Comment by virtualtam** [comment]
@ArthurHoaro we could create a Translation team on Github and/or setup [Transifex](https://www.transifex.com/) to encourage contributions, see related comments:
- https://github.com/shaarli/Shaarli/issues/121#issuecomment-237828973
- https://github.com/shaarli/Shaarli/issues/106#issuecomment-75052621

**Comment by nodiscc** [comment]
I second Transifex. I help maintaining https://www.transifex.com/youtube-dl-gui/youtube-dl-gui/ and so far it has been very successful, and pleasing to work with.

**Comment by buster-one** [comment]
> Did you use the generate_templates script?

Yeah several times. The script runs well but the line has not been generated.

> Do you want to be the maintainer of German translation, and be notified of new translations?

Yes a very good idea. It would be e pleasure to support this project in this way.

 Can not say anything about Transifex yet, but Poedit worked very well.

File: inc/languages/de/LC_MESSAGES/shaarli.po (hunk 2 above)
Comment (buster-one) [reply inc/languages/de/LC_MESSAGES/shaarli.po]: It's fine. Others also contribute to it.

**Comment by virtualtam** [comment]
@shaarli/translators team created! @buster-one, I've sent you an invitation so you can join it :)

**Comment by buster-one** [comment]
@virtualtam Invitation accepted. :)

File: inc/languages/de/LC_MESSAGES/shaarli.po (hunk 3 above)
Comment (buster-one) [reply inc/languages/de/LC_MESSAGES/shaarli.po]: was corrected as suggested

File: inc/languages/de/LC_MESSAGES/shaarli.po (hunk 1 above)
Comment (buster-one) [reply inc/languages/de/LC_MESSAGES/shaarli.po]: removed as suggested

**Comment by virtualtam** [comment]
Thanks, merged! :)

**Pull request merged**